sandbox/Settings.qml

```diff
@@ -0,0 +1,67 @@
+import QtQuick 2.14
+import QtQuick.Layouts 1.14
+import StatusQ.Core 0.1
+import StatusQ.Core.Theme 0.1
+import StatusQ.Components 0.1
+
+Column {
+    spacing: 12
+    width: 800
+    anchors.top:parent.top
```

Comment:
thats not good. If it possible use `anchors, width and height` when you use this object. Here should be just `natural sizes` like `implicitWidth` :)

Reply:
Hey @borismelnik  this is only a demonstration of how to use the StatusExpandableSettingsItem component, Should be okay right?